### PR TITLE
4 add avax bsc

### PR DESCRIPTION
**Pull request by Thorian1te**

### CODE REVIEW

File: src/main/api/ledger/address.ts [hunk 1]
```diff
@@ -58,13 +60,24 @@ export const getAddress = async ({
...
-            res = await getETHAddress({ transport, walletIndex, ethHdMode: hdMode })
+            res = await getETHAddress({ transport, walletIndex, evmHdMode: hdMode })
+          }
+          break
+        }
+        case AVAXChain: {
+          if (!isEvmHDMode(hdMode)) {
+            res = E.left({
+              errorId: LedgerErrorId.INVALID_ETH_DERIVATION_MODE,
+              msg: `Invaid 'EthHDMode' - needed for ETH to get Ledger address`
```
Comment (slack-coder) [inline src/main/api/ledger/address.ts:77]: 'Invaid' -> 'Invalid' ?

Comment (Thorian1te) [reply src/main/api/ledger/address.ts:77]: thanks 👍

**Review by Thorian1te — COMMENTED** [review]
looks good

**Review by GrassRootsCrypto — APPROVED** [review]
Good job, tested locally.

**Pull request merged**